help/faq/statfaq.md

```diff
@@ -1,82 +1,14 @@
-    activity levels of archive), there are as well many other access
-    points for these archives, including mirror sites and local
-    redistribution points (not to mention local photocopying, etc.). For
-    example, there are sites that simply download everything either for
-    local printing or local caching; so there's sometimes an automatic
-    lower bound on the raw number of accesses, which is not necessarily
-    a measure of readership. On the other hand, this distributed access
-    means as well that many papers may be significantly undercounted if
-    one takes into account only the centralized public distribution
-    points.
-    
-    There are as well other distorting factors. For example a paper's
-    initial submission may have some minor technical processing problem,
-    corrected by a quick replacement the next day, and hence have many
-    2nd requests for a viewable version, perhaps artificially doubling
-    the initial raw number of requests.
-    
-    In addition, it is impossible to distinguish between actual reading
-    and browsing or other spurious requests. For example, many
-    physicists tend to use these papers as a substitute for memory --
-    i.e. they know a paper with a particular equation, so get it just
-    for that equation then delete it (should that count as reading?).
-    
-    *So before publicizing any such statistics, all the caveats would
-    have to be clearly understood.*
-    
-    But then the very availability of statistics could have many
-    undesirable consequences. Some segment of submitters might find it
-    highly embarrassing to see how little read is (and consequently how
-    little impact has) their magnum opus on which they've expended so
-    much quality time and effort. Furthermore, we don't wish to
-    encourage people to try somehow to distort public statistics in
-    their favor either by self-generating large numbers of requests, or
-    by repeated replacement of their submission to generate multiple
-    repeat requests.
-    
-    It could be argued perhaps correctly that statistics may provide
-    some useful information at least on the *relative* popularity of
-    submissions, since the distributed access and other factors may be
-    subsumable into some overall scale factor. But even this information
-    is ambiguous in many cases, and publicizing, even when accurate,
-    could merely accentuate faddishness in fields already excessively
-    faddish.
-    
-    *Most significantly*, however, there is a strong philosophic reason
-    for not publicizing (or even saving) these statistics. When one
-    browses in a library it is very important (in fact legislated) that
-    big brother is not watching through a camera mounted on the wall;
-    for the benefit of readers it is very important to maintain in every
-    way possible this sense of freedom from monitoring in the electronic
-    realm.
+  - ### Why?

```

Comment:
I would get rid of this, because it sounds terse and leads to more questions. The answer above is adequate.

Reply:
This one was dropped in the latest go-round after discussing with Steinn.